src/lnp/presentation/encoding.rs

```diff
@@ -395,6 +395,24 @@ mod byte_strings {
         }
     }
 
+    impl LightningEncode for [u8; 3] {
```

Comment:
We need to remove this, since different `[u8; 3]`-based types may have different encoding requirements

Reply:
Separately implemented encoding for these types.